_episodes/13-looking-up-data.md

```diff
@@ -92,7 +92,7 @@ One of the most common ways of using the reconciliation option in OpenRefine is
 
 Other extensions are available to do reconciliation against local data such as csv files (see [http://okfnlabs.org/reconcile-csv/](http://okfnlabs.org/reconcile-csv/)) and maintained lists of values (see [http://okfnlabs.org/projects/nomenklatura/index.html](http://okfnlabs.org/projects/nomenklatura/index.html)).
 
-For more information on using Reconciliation services see [https://github.com/OpenRefine/OpenRefine/wiki/Reconciliation-Service-API](https://github.com/OpenRefine/OpenRefine/wiki/Reconciliation-Service-API)
+For more information on Reconciliation services see [https://reconciliation-api.github.io/census/](https://reconciliation-api.github.io/census/).
```

Comment:
The old wiki page indicates that the information is now at https://docs.openrefine.org/technical-reference/reconciliation-api so I think we should reference this rather than the census pages

Reply:
Right, however that page no longer contains any list of services, and is not actually about *using* Reconciliation services. I took another look at the available pages and changed it back to the original text with a link to https://docs.openrefine.org/manual/reconciling, which links to many useful resources. What do you think?